{"metadata":{"role":"symbol","externalID":"s:10ZendeskSDK0A5EventO20authenticationFailedyACs5Error_p_tcACmF","symbolKind":"case","roleHeading":"Case","modules":[{"name":"ZendeskSDK"}],"fragments":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"text":"authenticationFailed","kind":"identifier"},{"text":"(","kind":"text"},{"text":"error","kind":"externalParam"},{"text":": ","kind":"text"},{"text":"Error","preciseIdentifier":"s:s5ErrorP","kind":"typeIdentifier"},{"text":")","kind":"text"}],"title":"ZendeskEvent.authenticationFailed(error:)"},"kind":"symbol","primaryContentSections":[{"declarations":[{"platforms":["iOS"],"languages":["swift"],"tokens":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"text":"authenticationFailed","kind":"identifier"},{"text":"(","kind":"text"},{"text":"error","kind":"externalParam"},{"text":": ","kind":"text"},{"preciseIdentifier":"s:s5ErrorP","text":"Error","kind":"typeIdentifier"},{"text":")","kind":"text"}]}],"kind":"declarations"},{"parameters":[{"name":"error","content":[{"type":"paragraph","inlineContent":[{"type":"text","text":"The error that caused the authentication failure."}]}]}],"kind":"parameters"}],"sections":[],"abstract":[{"text":"Invoked when a rest call fails for authentication reasons.","type":"text"}],"hierarchy":{"paths":[["doc:\/\/com.zendesk.Zendesk\/documentation\/ZendeskSDK","doc:\/\/com.zendesk.Zendesk\/documentation\/ZendeskSDK\/ZendeskEvent"]]},"identifier":{"url":"doc:\/\/com.zendesk.Zendesk\/documentation\/ZendeskSDK\/ZendeskEvent\/authenticationFailed(error:)","interfaceLanguage":"swift"},"variants":[{"paths":["\/documentation\/zendesksdk\/zendeskevent\/authenticationfailed(error:)"],"traits":[{"interfaceLanguage":"swift"}]}],"schemaVersion":{"major":0,"patch":0,"minor":3},"references":{"doc://com.zendesk.Zendesk/documentation/ZendeskSDK/ZendeskEvent":{"identifier":"doc:\/\/com.zendesk.Zendesk\/documentation\/ZendeskSDK\/ZendeskEvent","url":"\/documentation\/zendesksdk\/zendeskevent","abstract":[{"type":"text","text":"The set of events that can be emitted from Zendesk SDK."}],"fragments":[{"text":"enum","kind":"keyword"},{"text":" ","kind":"text"},{"text":"ZendeskEvent","kind":"identifier"}],"navigatorTitle":[{"text":"ZendeskEvent","kind":"identifier"}],"type":"topic","role":"symbol","kind":"symbol","title":"ZendeskEvent"},"doc://com.zendesk.Zendesk/documentation/ZendeskSDK":{"title":"ZendeskSDK","identifier":"doc:\/\/com.zendesk.Zendesk\/documentation\/ZendeskSDK","abstract":[],"kind":"symbol","type":"topic","url":"\/documentation\/zendesksdk","role":"collection"},"doc://com.zendesk.Zendesk/documentation/ZendeskSDK/ZendeskEvent/authenticationFailed(error:)":{"role":"symbol","kind":"symbol","title":"ZendeskEvent.authenticationFailed(error:)","type":"topic","abstract":[{"text":"Invoked when a rest call fails for authentication reasons.","type":"text"}],"fragments":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"text":"authenticationFailed","kind":"identifier"},{"text":"(","kind":"text"},{"text":"error","kind":"externalParam"},{"text":": ","kind":"text"},{"text":"Error","preciseIdentifier":"s:s5ErrorP","kind":"typeIdentifier"},{"text":")","kind":"text"}],"identifier":"doc:\/\/com.zendesk.Zendesk\/documentation\/ZendeskSDK\/ZendeskEvent\/authenticationFailed(error:)","url":"\/documentation\/zendesksdk\/zendeskevent\/authenticationfailed(error:)"}}}